### PR TITLE
Add ability to create and update tiles

### PR DESCRIPTION
**Pull request by patrickberkeley**
Associated UI PR: https://github.com/AgilionApps/LaunchPad-UI/pull/6

//cc @alanpeabody

### CODE REVIEW

File: app/controllers/v1/tiles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class V1::TilesController < ApplicationController
+  def create
+    tile = Tile.new(create_params)
```
Comment (alanpeabody) [inline app/controllers/v1/tiles_controller.rb]: Can anyone create tiles on any project? Or should this be enforcing project_id is in current_user.projects?

File: app/controllers/v1/tiles_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    if tile.save
+      render json: tile, status: :created, serializer: TileSerializer
+    else
+      render_errors tile
+    end
+  end
+
+  def update
+    tile = Tile.find(params[:id])
```
Comment (alanpeabody) [inline app/controllers/v1/tiles_controller.rb]: Same comment on permissions here.

File: app/controllers/v1/tiles_controller.rb (hunk 1 above)
Comment (patrickberkeley) [inline app/controllers/v1/tiles_controller.rb]: Only people on the project's team can create tiles, but I don't have the project_people table setup yet. Do you think it's okay to hold off on permissions for this until I have that in?

Here's what I was thinking for it:

Comment (alanpeabody) [inline app/controllers/v1/tiles_controller.rb]: Okay, I think it is okay to wait, maybe add a TODO: comment here and a ticket so we don't miss it once those are in place.

**Comment by alanpeabody** [comment]
LGTM

File: app/controllers/v1/tiles_controller.rb (hunk 1 above)
Comment (patrickberkeley) [inline app/controllers/v1/tiles_controller.rb]: :+1: Done

**Pull request merged**